### PR TITLE
Receive events from Marathon & relay events to Consul.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@miltontony & @Rizziepit first PR ready for review. Bare bones relay of events from Marathon to Consul, ignores any staging flags at this point. Next PR addresses flags.

**Comment by smn** [comment]
Marathon events documented here: https://mesosphere.github.io/marathon/docs/event-bus.html
Consul HTTP API documented here: https://consul.io/docs/agent/http/agent.html

**Comment by smn** [comment]
@bruskiza also pinging you, ready for review.

**Comment by smn** [comment]
Oops, merged #3 into this one.

File: consular/main.py [hunk 1]
```diff
@@ -10,8 +19,72 @@ class Consular(object):
     def __init__(self, consul_endpoint, marathon_endpoint):
         self.consul_endpoint = consul_endpoint
         self.marathon_endpoint = marathon_endpoint
+        self.pool = HTTPConnectionPool(reactor, persistent=False)
+        self.event_dispatch = {
+            'status_update_event': self.handle_status_update_event,
+        }
+        self.unknown_event_handler = self.handle_unknown_event
```
Comment (Rizziepit) [inline consular/main.py]: I'm wondering why the rename here? I see `handle_unknown_event` being used below but not `unknown_event_handler`.

Comment (smn) [inline consular/main.py]: Good catch, thought I needed to mock this out for tests, turns out not needed.

**Comment by miltontony** [comment]
:+1:  looks good

**Comment by Rizziepit** [comment]
+1 from me

**Pull request merged**